app/assets/stylesheets/common/components/bookmark-menu.scss

```diff
@@ -0,0 +1,113 @@
+@keyframes slide {
+  0% {
+  }
```

Comment:
do you need this?

Reply:
Ah no this seems like it's from the old notice before we changed to toast, removing